src/SluggerHelper.php

```diff
@@ -13,6 +15,8 @@ public static function slug(string $string): string
     {
         self::$slugger ??= new AsciiSlugger();
 
+        $string = u($string)->snake()->toString();
```

Comment:
previous, something named `dryRun` was transformed to `dryrun`, now it's `dry-run`